mkdocs.yml

```diff
@@ -15,17 +15,20 @@ pages:
       - Sending Log Events:
         - Overview: logs/sending-log-events.md
         - Using Elasticsearch API: logs/index-events-via-elasticsearch-api.md
-        - Logagent: logs/logagent.md
-        - Logstash: logs/logstash.md
-        - Syslog:
-          - Overview: logs/syslog.md
-          - JSON Messages over Syslog: logs/json-messages-over-syslog.md
-          - rsyslog: logs/rsyslog.md
-          - syslogd: logs/syslogd.md
-          - syslog-ng: logs/syslog-ng.md
-          - Certificates: logs/certificates.md
-          - Authorizing IPs for Syslog: logs/authorizing-ips-for-syslog.md
-        - JSON: logs/json.md
+        - Sending Server Logs:
```

Comment:
Why *Server*?  Ah, you are probably trying to differentiate between Server and Containers.  Not sure if "Server" is the right word choice or if we even need a word there.

Reply:
I will remove the group. Then people see syslog  & co earlier.